tests/opf/test_io.py

```diff
@@ -58,10 +58,16 @@ def test_missing_field(self):
 class TestDatasets(unittest.TestCase):
     def test_load_opf_example(self):
         # Check that example cases can be loaded by name
-        for case in ["case9", "case14", "case57", "case118", "case300", "caseNY"]:
+        for case in [
```

Comment:
Just replacing all the test cases makes it hard to tell if these changes break anything that used to work (even if the new files are better). I would prefer the tests to be extended, rather than replaced.